### PR TITLE
[MIG] stock_lot_scrap: Migration to 11.0

### PR DESCRIPTION
**Pull request by cubells**
cc @Tecnativa

### CODE REVIEW

**Review by rousseldenis — APPROVED** [review]
Code review

File: stock_lot_scrap/readme/USAGE.rst [hunk 1]
```diff
@@ -0,0 +1,3 @@
+* Go to *Inventory > Inventory Control > Serial Numbers / Lots*, and enter in
```
Comment (chienandalu) [inline stock_lot_scrap/readme/USAGE.rst]: *Inventory > Master Data > Lots/Serial Numbers*

**Review by chienandalu — APPROVED** [review]
Tested ok in runbot. Just a minor comment below.

Also, it shouldn't be very hard to get full coverage, isn't it?

**Comment by pedrobaeza** [comment]
I have fixed README and add tests for the missing part, so we can merge.

**Pull request merged**